### PR TITLE
Added zstd package

### PR DESCRIPTION
**Pull request by JL2210**
Added zstd to Packages. Hint: Apt coming up soon.

### CODE REVIEW

File: packages/zstd.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+require 'package'
+
+class Zstd < Package
+  description 'Zstandard - Fast real-time compression algorithm'
+  homepage 'http://www.zstd.net'
+  version '1.3.4'
+  source_url 'https://github.com/facebook/zstd/archive/master.tar.gz'
```
Comment (uberhacker) [inline packages/zstd.rb]: We should use the versioned source at https://github.com/facebook/zstd/archive/v1.3.4.tar.gz.

File: packages/zstd.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  source_sha256 '7fba0070789a3c1e343fab4660958b73cc262110d2977589ba158b169d331ad6'
+  
+  def self.build
+    system 'make'
```
Comment (uberhacker) [inline packages/zstd.rb]: Is there a configure script?  If so, we will need the prefix and possibly the libdir abstracted.

File: packages/zstd.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  end
+
+  def self.install
+    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
```
Comment (uberhacker) [inline packages/zstd.rb]: If there is no configure script, you could add `PREFIX=#{CREW_PREFIX}` here.

**Comment by JL2210** [comment]
No configure script.

**Pull request merged**